spec/acceptance/appraisals_file_bundler_dsl_compatibility_spec.rb

```diff
@@ -58,6 +60,7 @@
         end
 
         group :breakfast do
+          gem 'rotten_egg', :remove
```

Comment:
Another API you might consider for this is:

```suggestion
          remove_gem 'rotten_egg'
```

I think this makes it a bit easier to parse visually, whereas with the `:remove` option you have to read to the end of the line to see whether the method is removing or adding something.

This also has the benefit of removing a conditional in the `gem` method.

Reply:
I do like how this reads a lot better. I originally thought about doing this, but was hesitant to implement it this way due to Appraisal looking like it's trying to not add to the Bundler DSL, which adding the `remove_gem` command would be doing.  Though, I think the readability is worth the addition, so I'll go ahead and get this change worked in, along with some documentation in the README about how to utilize this like @nickcharlton mentioned.

Reply:
Oh yeah, much nicer. Thanks for suggesting it, @composerinteralia!